src/test/java/uk/gov/hmcts/dts/fact/services/admin/AdminAuditServiceTest.java

```diff
@@ -0,0 +1,109 @@
+    private static final String TEST_LOCATION = "mosh court";
+    private static final String TEST_EMAIL = "kupo email";
+
+    @BeforeAll
+    static void beforeAll() {
+        AUDIT_DATA.add(new Audit("Test String", new AuditType(1, "test"),
+                               "data before", "data after",
+                               "some court", LocalDateTime.of(1000, 10, 10, 10, 10)));
+        AUDIT_DATA.add(new Audit("Test String 2", new AuditType(1, "test 2"),
+                               "data before 2", "data after 2",
+                               "some court", LocalDateTime.of(1000, 10, 10, 10, 10)));
+        AUDIT_DATA.get(0).setId(0);
+        AUDIT_DATA.get(1).setId(1);
+    }
+
+    @Test
+    void shouldGetAllAuditDataNoDateRange() {
+        when(auditRepository.findAllByLocationContainingAndUserEmailContaining(
+            TEST_LOCATION, TEST_EMAIL, PageRequest.of(0, 10))).thenReturn(new PageImpl<>(AUDIT_DATA));
+
+        final List<uk.gov.hmcts.dts.fact.model.admin.Audit> results =
+            adminAuditService.getAllAuditData(0, 10,
+                                              Optional.of(TEST_LOCATION), Optional.of(TEST_EMAIL),
+                                              Optional.empty(), Optional.empty());
+
+        verify(auditRepository, atLeastOnce()).findAllByLocationContainingAndUserEmailContaining(
+            TEST_LOCATION, TEST_EMAIL, PageRequest.of(0, 10));
+        assertThat(results).hasSize(2);
+        assertThat(results.get(0)).isEqualTo(new uk.gov.hmcts.dts.fact.model.admin.Audit(AUDIT_DATA.get(0)));
+        assertThat(results.get(1)).isEqualTo(new uk.gov.hmcts.dts.fact.model.admin.Audit(AUDIT_DATA.get(1)));
+    }
+
+    @Test
+    void shouldGetAllAuditDataWithDateRange() {
+        final LocalDateTime dateFrom = LocalDateTime.of(1000, 10, 10, 10, 10);
+        final LocalDateTime dateTo = LocalDateTime.of(1000, 12, 10, 10, 10);
+        when(auditRepository.findAllByLocationContainingAndUserEmailContainingAndCreationTimeBetween(
+            TEST_LOCATION, TEST_EMAIL, dateFrom, dateTo,
+            PageRequest.of(0, 10))).thenReturn(new PageImpl<>(AUDIT_DATA));
+
+        final List<uk.gov.hmcts.dts.fact.model.admin.Audit> results =
+            adminAuditService.getAllAuditData(0, 10,
+                                              Optional.of(TEST_LOCATION), Optional.of(TEST_EMAIL),
+                                              Optional.of(dateFrom), Optional.of(dateTo));
+
+        verify(auditRepository, atLeastOnce()).findAllByLocationContainingAndUserEmailContainingAndCreationTimeBetween(
+            TEST_LOCATION, TEST_EMAIL, dateFrom, dateTo, PageRequest.of(0, 10));
+        assertThat(results).hasSize(2);
+        assertThat(results.get(0)).isEqualTo(new uk.gov.hmcts.dts.fact.model.admin.Audit(AUDIT_DATA.get(0)));
+        assertThat(results.get(1)).isEqualTo(new uk.gov.hmcts.dts.fact.model.admin.Audit(AUDIT_DATA.get(1)));
```

Comment:
Same comment as previous test class - it might test the logic more to formulate a query that should return part of the data.